classes/announcement/PKPAnnouncementDAO.inc.php

```diff
@@ -338,7 +338,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
 			FROM announcements
 			WHERE assoc_type = ?
 				AND assoc_id = ?
-				AND (date_expire IS NULL OR date_expire > CURRENT_DATE)
+				AND (date_expire IS NULL OR date_expire >= CURRENT_DATE)
```

Comment:
What does date_expire have to do with the publish date? I am not sure if I followed your fix, but it seems that you changed the way the expire date is working, getting announcements that would be expired if the equal operator was not used.